src/Tools/BuildValidator/BuildConstructor.cs

```diff
@@ -48,8 +51,8 @@ public Compilation CreateCompilation(MetadataReader metadataReader, string name)
             {
                 var compilation = language switch
                 {
-                    LanguageNames.CSharp => CreateCSharpCompilation(pdbReader, name),
-                    LanguageNames.VisualBasic => CreateVisualBasicCompilation(pdbReader, name),
+                    LanguageNames.CSharp => CreateCSharpCompilationAsync(optionsReader, name),
+                    LanguageNames.VisualBasic => CreateVisualBasicCompilationAsync(optionsReader, name),
                     _ => throw new InvalidDataException($"{language} is not a known language")
```

Comment:
This should be `Task.FromException` vs a direct `throw`. May be easier to make the method `async`

Reply:
Fixed